lightning/src/routing/router.rs

```diff
@@ -2580,7 +2580,7 @@ where L::Target: Logger {
 			let maybe_announced_channel = if let CandidateRouteHop::PublicHop { .. } = hop.candidate {
 				// If we sourced the hop from the graph we're sure the target node is announced.
 				true
-			} else if let CandidateRouteHop::FirstHop { details } = hop.candidate {
+			} else if let CandidateRouteHop::FirstHop { details, .. } = hop.candidate {
```

Comment:
Please apply this to the commit where it was missed using an interactive rebase or insert a fixup commit after said commit.